crates/matrix-sdk-sqlite/src/event_cache_store.rs

```diff
@@ -1899,6 +1898,25 @@ mod tests {
 
         let chunks = store.load_all_chunks(room_id).await.unwrap();
         assert!(chunks.is_empty());
+
+        // It's okay to re-insert a past event.
```

Comment:
Do we want to add a `TODO: it's not okay actually!!` or something like this?

Reply:
It's actually okay to do that: if we remove an event because it is duplicated, then it is re-inserted, it's perfectly normal, right?

Reply:
Ah, I didn't notice it was removed in the middle; so yes that's fine then!